spec/classes/puppet_init_spec.rb

```diff
@@ -80,6 +80,21 @@
         it { should contain_puppet__config__main('ca_port').with_value(8140) }
       end
 
+      describe 'with server_ssl_dir' do
+        let :params do {
+          :server => true,
+          :server_ssl_dir => '/etc/custom/ssl/dir',
+        } end
+
+        it {
+          should contain_file('/etc/custom/puppetserver/conf.d/webserver.conf')
```

Comment:
Could you modify this path to the default one? Then the tests should at least find the right file and show a diff

Reply:
Less easy than expected due to supported OSes (ie. default path depends on it) but now done.